support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -174,12 +183,19 @@ export interface PaymentTsAndCsProps {
 	thresholdAmount?: number;
 }
 
+const rightReservation = `Offer subject to availability. Guardian News and Media Ltd ("GNM") reserves the right to withdraw this promotion at any time. `;
+function weeklyTsAndCs(isWeeklyGift?: boolean, promotion?: Promotion) {
+	return (
+		<>
+			{isWeeklyGift && !promotion && <div>{rightReservation}</div>}
```

Comment:
Is it right that we show the `rightReservation` copy which includes `reserves the right to withdraw this promotion at any time` when there isn't a promotion?